doc/manual/manual.tex

```diff
@@ -8485,23 +8488,249 @@ \subsubsection{The van Keken thermochemical composition benchmark}
+
+However, we can compare the required run time for a VOF computation to that for a DG computation.
+We note that the use of the VOF advection algorithm significantly reduces the required computation time in all cases, frequently requiring less than half the time required by the DG computational field.
+
+We now examine the RMS velocity data shown in Figure~\ref{fig:vof-vk-1}.
+Other than for the case of 5 levels of uniform global refinement, the curves for the RMS 
+velocities for $6$, $7$, $8$, $9$ and $10$ levels of refinement in Figure~\ref{fig:vof-vk-1} 
+are nearly indistinguishable.
+
+Upon examining the solution at the final time, we note that the general structure of the 
+solution shown in Figure~\ref{fig:VOF_van_Keken-02} matches the form and the general structure 
+found in other versions of this benchmark such as the fourth image on the right in 
+Figure~\ref{fig:vk-1}.
+We also note that the differences in the shape of the interface based on a single refinement 
+as shown in Figure~\ref{fig:VOF_van_Keken-02} are minor, although still slightly visible.
+This is to be expected as refinement is a perturbation of the initial condition at a smaller 
+wave length.
+
+\begin{table}
+  \centering
+   \begin{tabular}{|c|c|c|c|}
+     \hline
+      Global Refinement & Number of Processors & VOF & DG \\
+      \hline
+\phantom{1}5 & \phantom{1}4 &\phantom{6}1.33 minutes\phantom{hours} &\phantom{1}2.57 minutes  \\
+\phantom{1}6 & \phantom{1}4 &\phantom{1}8.51 minutes\phantom{hours} & 19.5\phantom{7} minutes \\
+\phantom{1}7 & \phantom{1}4 &\phantom{1}1.15 hours\phantom{minutes} & \phantom{1}2.49 hours\phantom{tes} \\
+\phantom{1}8 & \phantom{1}4 &\phantom{1}8.53 hours\phantom{minutes} & 19.6 hours\phantom{tes} \\
+\phantom{1}9 & 16           &         16.30 hours\phantom{minutes}  & \phantom{1}2.72 days\phantom{tes} \\
+          10 & 16           & \phantom{1}5.17 days\phantom{minutess}&>6.00 days\phantom{tess} \\
+            \hline
+        \end{tabular}
+        \caption{\it Comparison of runtimes for the van Keken problem with VOF and and a DG computational field, in which the initial conditions for DG smoothed as described in section~\ref{paragraph:van-keken compositional fields} above.
+            The times shown are for the full computation, ending at $t_{end} = 2000$ with a CFL
+            number of $\sigma=0.5$ in both cases. 
+            All of these computations were made with ASPECT version 2.2.0-pre (master, ef542ecc2) in release mode on the Peloton2 cluster at U.C.~Davis.
+            We note that the change in the CFL number $\sigma$ and the differing
+            choice of cluster makes a direct quantitative comparison between this
+            table and table~\ref{tab:runtime-table} invalid due to too many
+            confounding factors.
+        }
+        \label{tab:vof-runtime-comparison-table}
+\end{table}
+
+The consistency of the results shown here differs noticeably from the behavior of the  
```

Comment:
In this paragraph and some above you are starting multiple sentences on new lines. Is this intentional for the purposes of readability?

Reply:
Yes, it is intentional, but the primary purpose was to attempt to make diffs cleaner, rather than the primary intent being tied to the readability of the latex file itself.